plugins/catalog-react/src/components/EntityDisplayName/EntityDisplayName.tsx

```diff
@@ -77,7 +86,9 @@ export const EntityDisplayName = (
   );
 
   // The innermost "body" content
-  let content = <>{primaryTitle}</>;
+  let content = (
```

Comment:
Just curiosity, how does that look in mobile screens?

Reply:
Mobile View will look like this 
<img width="406" alt="Screenshot 2024-10-18 at 7 44 38 PM" src="https://github.com/user-attachments/assets/5ffe09db-e771-459d-bac5-efcdfe1f8572">